crates/rolldown_binding/src/bundler.rs

```diff
@@ -19,6 +18,13 @@ use napi_derive::napi;
 use rolldown::Bundler as NativeBundler;
 use rolldown_error::{BuildDiagnostic, BuildResult, DiagnosticOptions};
 
+#[napi(object, object_to_js = false)]
+pub struct BindingBundlerOption {
```

Comment:
```suggestion
pub struct BindingBundlerOptions {
```

Plural would be better.